### PR TITLE
PB-118: Take into account the GET key limit as well on bbox searches

### PR DESCRIPTION
**Pull request by rebert**

### CODE REVIEW

File: app/search.py [hunk 1]
```diff
@@ -256,7 +256,7 @@ def _swiss_search(self):  # pylint: disable=too-many-branches, too-many-statemen
 
             wildcard_results = results[0].get('matches', [])
             merged_results = []
-            if len(results) == 2:
+            if len(results) == 1:
```
Comment (ltclm) [inline app/search.py:259]: sure about that?
are the e2e tests happy with it?

File: app/search.py [hunk 2]
```diff
@@ -197,7 +197,8 @@ def _swiss_search(self):  # pylint: disable=too-many-branches, too-many-statemen
             coords = self._get_geoanchor_from_bbox()
             self.sphinx.SetGeoAnchor('lat', 'lon', coords[1], coords[0])  # pylint: disable=unsubscriptable-object
             self.sphinx.SetSortMode(sphinxapi.SPH_SORT_EXTENDED, '@geodist ASC')
-            limit = self.BBOX_SEARCH_LIMIT
+            limit = self.limit if self.limit and \
```
Comment (ltclm) [inline app/search.py:200]: cool, thank you :+1:

**Review by ltclm — APPROVED** [review]

File: app/search.py (hunk 1 above)
Comment (rebert) [reply app/search.py:259]: no, I will investigate - it is more the merge of both results that seems to have a bug...

**Comment by rebert** [comment]
e2e-tests passed successfully on this pr

File: app/search.py (hunk 2 above)
Comment (ltshb) [reply app/search.py:200]: could be simplified by `limit = min(self.limit, self.BBOX_SEARCH_LIMT)` Of course for this to work both parameter needs to be numbers

**Review by ltshb — APPROVED** [review]

**Pull request merged**